### ISSUE TITLE
Set up Analytics

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#701]
**Track Quarterly (on website):**
- Cases Published 
- Methods Published 
- Organizations Published 
- User Accounts Created
- Total Sessions
- % New Visitors
- Average Session Duration
- Average Bounce Rate
- Geography of Users (% of total, by country)

**Plus social media analytics.**

We discussed hiring a digital campaigner or someone to set up an effective strategy for us. I sent the following note to Alex Samuel on December 7, 2017 but have not heard back. 

Hi Alex,
It's been a long time, I hope you're doing well!
I'm still with Participedia and we're working towards launching the redesigned website (test site: participedia.xyz). We'd like to set up an analytics strategy, and we thought you might have suggestions of a firm or individual that we could hire who has expertise in this area, as well as digital campaigning / social media strategy. We're thinking about an initial investment in a professional to set us up with a solid strategy that we can take forward and manage internally. What are your thoughts on this, and do you have any recommendations?
Thanks so much,
Jesi

@dethe @plscully what do you think the next steps should be here? Do we know anyone else we can reach out to, or should I start researching companies who do this? We should have something set up for tracking at least basic google analytics for the launch, don't you think? Thanks!

**Comment by scottofletcher** [comment]
I'd be happy to take this on - I've been handling the social media for almost two years now (really one of my favourite parts of my work!) and I think I've got the hang of Google Analytics - @plscully thoughts? I'd be fine with one or the other (if not both) with a preference for social media. I'm handling the social media/communications side of another organization so will be able to carry over any teachings/experience to PPedia.

**Comment by jesicarson** [comment]
@scottofletcher Yes, I think ideally after we're all set up you would continue to create and post social content as well as manage the quarterly reporting for social and site usage. 

It might be also be worthwhile to hire a professional service to assess what we're currently analyzing and make sure we're capturing everything we need to get a clear picture. They might also offer strategies we haven't thought of to continue to grow our following. 

@dethe This issue also also needs to cover the technical side of setting up analytics for the new website. Could we get the basics set up before team launch? Do you know anyone who could do this for us if you and Andrea are too busy/ it's not your speciality?
